tests/test_bloommatcher.py

```diff
@@ -70,6 +73,28 @@ def test_dice_4_c(self):
 
         self.assertEqual(result, 0.0)
 
+# Generate bit arrays that are combinations of words 0, 1, 2^63, 2^64 - 1
+# of various lengths between 1 and 65 words.
+def test_dicecoeff():
```

Comment:
Nitpick: the comments should be the [Python docstring](https://en.wikipedia.org/wiki/Docstring#Python).